robot/web/web_test.go

```diff
@@ -696,15 +698,24 @@ func TestWebWithStreams(t *testing.T) {
 	robot.MockResourcesFromMap(rs)
 	updateable, ok := svc.(resource.Updateable)
 	test.That(t, ok, test.ShouldBeTrue)
-	err = updateable.Update(ctx, rs)
+	err = updateable.Update(context.Background(), rs)
```

Comment:
Changing `ctx` to `context.Background()` forces the `updateable` to rely on its internal `cancelFuncs` to preempt the  goroutine started by `Update`.

Reply:
> LGTM! Sorry for making you undo and redo work!

No worries. My misunderstanding.